test/serving.bash

```diff
@@ -61,15 +61,9 @@ function upstream_knative_serving_e2e_and_conformance_tests {
   fi
 
   SYSTEM_NAMESPACE=knative-serving go_test_e2e -tags=e2e -timeout=30m -parallel=$parallel \
-    ./test/e2e ./test/conformance/api/v1/... ./test/conformance/runtime/... \
+    ./test/e2e ./test/conformance/api/... ./test/conformance/runtime/... \
```

Comment:
The `--customdomain` is not passed anywhere. Is it supposed to be?

Reply:
No, when we specify the `--customdomain`, the specified domain needs to be "resolvable". And it is difficult (=needs extra cost) to set up the second domain for CI :disappointed:

So I think it is OK to use the default hard-corded `example.org` with spoof client rather than setting up the resolable domain.

Reply:
OK. Thanks!